rfcs/new-coc-and-moderation.md

```diff
@@ -0,0 +1,92 @@
+# RFC - New Code of Conduct and Moderation Process
+
+So far, Ayo.js has defaulted to using [Node.js' current Code of
+Conduct](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md), which is
+the standard Contributor Covenant. Furthermore, it has its own [Moderation
+Policy](https://github.com/nodejs/TSC/blob/master/Moderation-Policy.md). This
+RFC is a proposal to change the base Code of Conduct for the project, as well as
+change and streamline the moderation policy.
+
+### Moderation Policy Values
+
+This RFC aims to change priorities surrounding moderation a bit by changing the
+values the Ayo.js project's moderation process will operate based on and then
+applying those values into new policy. Roughly stated, the contrasting values
+are such:
+
+1. **Moderation is About Empathy** - seeing moderation processes as ways to
+"punish" bad behavior results in fear, paranoia, and gets in the way of peaceful
```

Comment:
We may want to replace or remove paranoia here. I know there's a push to try and limit its use as a descriptive term for mental health, and I think we'd prefer to err on the side of caution here, particularly if we can still get the point across with better wording.

Reply:
ah! yes, I agree. :) Thank you!